### PR TITLE
[bugfix] remove redundant and harmful metric appending

### PR DESCRIPTION
**Pull request by gameofdimension**
this line is unnecessary can cause error to metric reduce

### CODE REVIEW

File: verl/workers/actor/dp_actor.py [hunk 1]
```diff
@@ -325,6 +325,5 @@ def update_policy(self, data: DataProto):
 
                 grad_norm = self._optimizer_step()
                 data = {'actor/grad_norm': grad_norm.detach().item()}
-            append_to_dict(metrics, data)
```
Comment (hiyouga) [inline verl/workers/actor/dp_actor.py:328]: I think we can add indent to this line. The grad norm should be appended to the metrics
